pkg/authn/claims_test.go

```
@@ -0,0 +1,138 @@
+	require.NoError(t, err)
+	return strings.TrimSpace(out.String())
+}
+
+func getLatestVersion(t *testing.T) semver.Version {
+	tag := getLatestTag(t)
+	v, err := semver.NewVersion(tag)
+	require.NoError(t, err)
+
+	return *v
+}
+func buildFactoryWithVersion(
```

golangci-lint (1.62.2)

func `buildFactoryWithVersion` is unused

(unused)